### PR TITLE
Use mailtrap by default

### PR DESCRIPTION
**Pull request by jeppester**
To make initial project setup easier.

Asana: https://app.asana.com/0/1142794766483633/1202242176448729

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -59,16 +61,29 @@
...
+  if ENV.key?("SENDINBLUE_USERNAME")
+    config.action_mailer.smtp_settings = {
+      address: "smtp-relay.sendinblue.com",
+      port: 587,
+      user_name: ENV.fetch("SENDINBLUE_EMAIL"),
+      password: ENV.fetch("SENDINBLUE_PASSWORD"),
+      authentication: "login",
+      enable_starttls_auto: true
+    }
+  elsif ENV.key?("MAILTRAP_API_TOKEN")
```
Comment (jeppester) [inline config/environments/production.rb:75]: I would have preferred the app to break on boot when both of the keys are missing, but that unfortunately breaks our CI due to the asset precompilation step leading the production env settings.

File: config/environments/production.rb [hunk 2]
```diff
@@ -59,16 +61,29 @@
...
+    response = URI.parse("https:///api/v1/inboxes.json?api_token=#{ENV.fetch('MAILTRAP_API_TOKEN')}").read
+    first_inbox = JSON.parse(response)[0]
```
Comment (hcarreras) [inline config/environments/production.rb:77]: Smart. Is this from the documentation?

**Review by hcarreras — APPROVED** [review]
I like the idea, and I approve of mailtrap.io 
It would also stay that in Staging Mailtrap could stay (although it would make a difference between staging and production, it's quite confusing to receive an email from a Staging environment)

File: config/environments/production.rb (hunk 2 above)
Comment (jeppester) [reply config/environments/production.rb:77]: Yes, but I altered it slightly to not require a third party gem.

**Comment by jeppester** [comment]
> I like the idea, and I approve of mailtrap.io It would also stay that in Staging Mailtrap could stay (although it would make a difference between staging and production, it's quite confusing to receive an email from a Staging environment)

I have changed the text slightly to include that sendinblue is usually set up only for the production env.

**Pull request merged**